### PR TITLE
Dark Mode : Permission flow

### PR DESCRIPTION
**Pull request by GuillaumeRx**
## Explanation

Switch all `--black-100` occurence to use `--color-text-default` and fix some permission flow colors

## More information

* Fixes #13719

## Screenshots/Screencaps

dark
light

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]

File: ui/pages/permissions-connect/redirect/index.scss [hunk 1]
```diff
@@ -31,7 +31,7 @@
   &__check {
     width: 40px;
     height: 40px;
-    background: white url("/images/permissions-check.svg") no-repeat;
+    background: var(--color-background-default) url("/images/permissions-check.svg") no-repeat;
```
Comment (georgewrmarshall) [inline ui/pages/permissions-connect/redirect/index.scss:34]: Can I be a pain and request this instance be updated to use font-awesome icon. So remove this
```suggestion
```

And updated this instance
Screen Shot 2022-03-21 at 10 43 46 AM

To 
```
<i className="fa fa-check permissions-redirect__check" />
```

You may need to update the color and the font-size in the css to match what it currently looks like

**Review by georgewrmarshall — COMMENTED** [review]
Looking great!! Made one suggestion

File: ui/pages/permissions-connect/redirect/index.scss (hunk 1 above)
Comment (GuillaumeRx) [reply ui/pages/permissions-connect/redirect/index.scss:34]: Yes, will update :)

**Review by darkwing — DISMISSED** [review]

**Review by georgewrmarshall — DISMISSED** [review]
Amazing work! Love the added story too!

**Review by georgewrmarshall — DISMISSED** [review]

**Review by darkwing — DISMISSED** [review]

**Comment by darkwing** [comment]
Needs a rebase!

**Review by georgewrmarshall — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**